README.md

```diff
@@ -63,6 +63,12 @@ bin/cake migrations mark_migrated 20150417223600
 # Since Migrations 1.3.1, a new `all` special value for the version argumentwas added.
 # The following will mark all migrations found as migrated.
 bin/cake migrations mark_migrated all
+
+# The following option `--target` will try to mark the migrations from beginning to the given version (including it)
+bin/cake migrations mark_migrated 20150417223600 --target
+
+# You can use the option `--exclude` along with `--target` and it will try to mark the migrations from beginning until the given version (excluding it)
+bin/cake migrations mark_migrated 20150417223600 --target --exclude
```

Comment:
The current way of doing it would need to still function as is (but possibly show an deprecation message).

As for marking only a single migration as migrated, perhaps a `--only` option is needed?